### DISCUSSION TITLE
FAQ - What is the maximum capacity of a Hubs room?

### DISCUSSION THREAD

**Discussion by emclaren** [mozilla/hubs#2680] [General Questions & Discussion] [answered]
I am organizing an event in Hubs, what is the maximum number of users that can be together inside of a Hubs Space?

**Comment by emclaren** [comment, marked as answer]
We recommend a maximum of 25 in-room participants. This capacity is the default because low power devices, such as mobile phones and standalone VR headsets, may begin experiencing performance issues in busier rooms. You can adjust the room capacity in the room settings menu (up to 50).

Once a room is full, additional users (for example, participant #26 and beyond) can still participate by watching from the lobby. Users in the lobby can see what's happening in the room and interact via text chat; however, they are not represented as avatars in the room and their mic is not active. There is no hard limit on lobby capacity, however, performance may decrease once there are over one hundred individuals in the lobby.

If you want to scale Hubs for a larger event we recommend splitting crowds into multiple rooms with smaller groups, or live streaming the event, depending on the use case.

**Reply by bemyevent** [reply]
And what about aws cloud ? 25 or More users in the same room ?

**Reply by InfiniteLee** [reply]
You can scale beyond 25 users in a single room with Hubs Cloud (estimated limits based on instance types used available [here](https://hubs.mozilla.com/docs/hubs-cloud-aws-estimated-ccu-limits.html)), however your experience will be limited by what type of devices are used to enter. For mobile, for example, beyond 25 users or so will cause performance issues.
